### PR TITLE
CEDS-2407 - Implement TTL on the SFUS

### PR DESCRIPTION
**Pull request by SteveSugden**
Replaced uk.gov.hmrc.http.cache.client.CacheMap with a Mongo repository storing an "Answers" model with a TTL of 60 minutes.

Minimal refactoring of unit tests so that they continue to pass.  Acceptance tests pass without any changes.

Confirmed locally that cds-file-upload-frontend.answers collection is cleared after the configured TTL period.

### CODE REVIEW

File: app/connectors/AnswersConnector.scala [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import repositories.AnswersRepository
+import uk.gov.hmrc.time.DateTimeUtils
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class AnswersConnector @Inject()(val repository: AnswersRepository)(implicit ec: ExecutionContext) {
+
+  private val logger = Logger(this.getClass)
+
+  def removeByEori(eori: String): Future[Unit] = repository.remove("eori" -> eori).filter(_.ok).map(_ => (): Unit)
```
Comment (PatrykRudnicki) [inline app/connectors/AnswersConnector.scala:32]: Is `filter(_.ok)` necessary? You're mapping this to Unit, so there is no reason to use filter.

File: test/base/SpecBase.scala [hunk 2]
```diff
@@ -29,7 +31,7 @@ import play.api.test.FakeRequest
 import uk.gov.hmrc.http.HeaderCarrier
 import utils.Injector
 
-import scala.concurrent.ExecutionContext
```
Comment (PatrykRudnicki) [inline test/base/SpecBase.scala:32]: Probably unused import.

What about lines 21,22? probably the same

File: test/base/SpecBase.scala [hunk 3]
```diff
@@ -29,7 +31,7 @@ import play.api.test.FakeRequest
...
+import scala.concurrent.{ExecutionContext, Future}
```
Comment (PatrykRudnicki) [inline test/base/SpecBase.scala:34]: Probably unused import.

What about lines 21,22? probably the same

File: app/connectors/AnswersConnector.scala (hunk 1 above)
Comment (SteveSugden) [reply app/connectors/AnswersConnector.scala:32]: No idea.  It's what we do in movements FE CacheRepository.

File: test/base/SpecBase.scala (hunk 3 above)
Comment (SteveSugden) [reply test/base/SpecBase.scala:34]: Good spot.

**Pull request merged**